internal/config/types.go

```diff
@@ -104,6 +104,15 @@ type Config struct {
 	Connect       *Connect    `toml:"connect,omitempty" json:"connect,omitempty"`
 }
 
+func (c *Config) HasSecret(secret string) bool {
+	for _, s := range c.Secrets {
+		if s == secret {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
It would be good to have tests for this new method, even if it is a fairly simple method, it's nice to build confidence for future changes.

Reply:
Added some unit tests around this in https://github.com/posit-dev/publisher/pull/2273/commits/6feea3a03a050a1547ed043b604bcba7438d7353